addons/meme.py

```diff
@@ -0,0 +1,29 @@
+import discord
+import random
+from discord.ext import commands
+
+
+class Meme:
+    """
+    Meme commands.
+    """
+
+    def __init__(self, bot):
+        self.bot = bot
+        print('Addon "{}" loaded'.format(self.__class__.__name__))
+
+    @commands.has_permissions(kick_members=True)
```

Comment:
Why not use the hop role when we already define it? Kick_member means that they need kick permissions and as far as I know some staff don't have it - https://github.com/916253/Kurisu-Reswitched/blob/master/run.py#L136

Reply:
Or just remove permission checks on the meme commands

Reply:
Apparently I fell asleep when I was typing a response to this yesterday.

@tumGER:

I used the same perms as the commands these are based on.

Discord.py, by default, only allows checking for roles by their names. I don't think that this is a secure approach. Theoretically we could do our own check, like [I did here](https://github.com/AkBKukU/discord-bot/blob/master/cogs/moderation.py#L9-L12) ([also see here](https://github.com/AkBKukU/discord-bot/blob/master/config.py.template#L33-L38)), but that'd be too much overhead for a meme command.

@misson20000:

I considered that, but it's likely for people to abuse/spam it so I didn't remove the checks. Perhaps I could only allow it in #off-topic, but again, too much overhead.